libs/blocks/table/table.js

```diff
@@ -0,0 +1,430 @@
+  headingCols.forEach((col) => {
+    if (col.innerHTML) {
+      const elements = col.children;
+      if (!elements.length) {
+        col.innerHTML = `<p class="heading-title">${col.innerHTML}</p>`;
+      } else {
+        let textStartIndex = 0;
+        if (elements[0]?.querySelector('img')) {
+          textStartIndex += 1;
+        }
+        elements[textStartIndex]?.classList.add('heading-title');
+
+        if (elements[textStartIndex + 1]) {
+          elements[textStartIndex + 1].classList.add('pricing');
+        }
+
+        decorateButtons(col, 'button-l');
+
+        const buttonsWrapper = createTag('div', { class: 'buttons-wrapper' });
+        col.append(buttonsWrapper);
+        const buttons = col.querySelectorAll('.con-button');
+
+        buttons.forEach((btn) => {
+          const btnWrapper = btn.closest('P');
+          buttonsWrapper.append(btnWrapper);
+        });
+      }
+    }
+  });
+}
+
+function handleHighlight(table) {
+  const isHighlightTable = table.classList.contains('highlight');
+  const firstRow = table.querySelector('.row-1');
+  const firstRowCols = firstRow.querySelectorAll('.col');
+  const secondRow = table.querySelector('.row-2');
+  const secondRowCols = secondRow.querySelectorAll('.col');
+  let headingCols = null;
+
+  if (isHighlightTable) {
+    firstRow.classList.add('row-highlight');
+    firstRowCols.forEach((e) => e.classList.add('col-highlight'));
+    secondRow.classList.add('row-heading');
+    secondRowCols.forEach((e) => e.classList.add('col-heading'));
+    headingCols = secondRowCols;
+
+    firstRowCols.forEach((col, i) => {
+      const hasText = headingCols[i]?.innerText && col.innerText;
+      if (hasText) {
+        headingCols[i].classList.add('no-rounded');
+      } else if (!headingCols[i]?.innerText) {
+        col.classList.add('hidden');
+        headingCols[i]?.classList.add('hidden');
+      } else {
+        col.classList.add('hidden');
+        if (headingCols[i - 1] && !headingCols[i - 1]?.innerText) {
+          headingCols[i]?.classList.add('top-left-rounded');
+        }
+      }
+    });
+  } else {
+    headingCols = firstRowCols;
+    firstRow.classList.add('row-heading');
+
+    headingCols.forEach((e, i) => {
+      e.classList.add('col-heading');
+
+      if (e.innerText) {
+        if (headingCols[i - 1] && !headingCols[i - 1].innerText) {
+          e.classList.add('top-left-rounded');
+        }
+      } else {
+        e.classList.add('hidden');
+      }
+    });
+  }
+
+  handleHeading(headingCols);
+  const tableHighlightLoadedEvent = new Event('milo:table:highlight:loaded');
+  window.dispatchEvent(tableHighlightLoadedEvent);
+}
+
+function handleExpand(e) {
+  const sectionHead = e.closest('.row');
+  let nextElement = sectionHead.nextElementSibling;
+  const expanded = e.getAttribute('aria-expanded') === 'false';
+  e.setAttribute('aria-expanded', expanded.toString());
+  while (nextElement && !nextElement.classList.contains('devider')) {
```

Comment:
typo - should be `divider`

I think you should use consts for the classes that are used multiple times